run_wml_tests

```diff
@@ -202,6 +251,8 @@ if __name__ == '__main__':
         help="New timer value to use for batched tests, instead of 300s as default.")
     ap.add_argument("-bd", "--batch-disable", action="store_true",
         help="Disable test batching, may be useful if debugging a small subset of tests.")
```

Comment:
Is the `--batch-disable` option still useful, could it be done with `--batch-max=1` instead?

Reply:
I was thinking about that too... I think `--batch-disable` is slightly more optimal than `--batch-max=1` with the current code, but that's easily fixed. I was also considering keeping `--batch-disable` but making it an alias for `--batch-max=1` instead of a separate flag.